bin/prepare.js

```diff
@@ -112,7 +111,7 @@ async function prepare () {
             logWithIndent('Writing assigned urls / ports / dbs to app\'s .env', 2)
             const env = {
                 DATABASE_URL: `${LOCAL_PG_DB_PREFIX}/${app.pgName}`,
-                REDIS_URL: `${LOCAL_REDIS_DB_PREFIX}/${app.redisIndex}`,
```

Comment:
Are you sure you want to change that in this PR?

Reply:
I'm not completely sure